Makefile

```
@@ -1,5 +1,5 @@
 name ?= goldpinger
-version ?= 2.0.2
+version ?= v2.1.0
```

Can we keep the original naming convention, or is this a go module requirement?

```suggestion
version ?= 2.1.0
```

---

Sadly it's a requirement. From https://github.com/golang/go/wiki/Modules#modules

> Modules must be semantically versioned according to semver, usually in the form v(major).(minor).(patch), such as v0.1.0, v1.2.3, or v1.5.0-rc.1. The leading v is required. If using Git, tag released commits with their versions. Public and private module repositories and proxies are becoming available (see FAQ below).